### ISSUE TITLE
data and basis_data  in file  03d_cal_cf_weights.sh 

### ISSUE THREAD

**Issue by Note-Liu** [wangchi95/CF-Font#6]
I would like to know if the data and basis_data  in file  03d_cal_cf_weights.sh uses clustering data (subset ) or the entire training set?

**Comment by wangchi95** [comment]
In terms of effect, the more characters used, the better (the more accurate the weights obtained). 

However, for the test set, we can only get the weight through the few-shot subset `FS16`. To simplify the process, we use the same few-shot subset for both the training and test sets.

**Comment by Note-Liu** [comment]
> Reference in

Understand! Thanks  a lot!
